pkg/option/config_test.go

```diff
@@ -726,6 +726,105 @@ func Test_populateNodePortRange(t *testing.T) {
 	}
 }
 
+func Test_populateNodePortAddresses(t *testing.T) {
```

Comment:
IPv4-mapped IPv6 addresses might be an interesting case in here though I don't know if that even makes sense for the feature ;-)

Reply:
In theory, this should never happen, as they will be separated by this point.